README.md

```diff
@@ -26,33 +27,49 @@ Add a dependency using your favorite build tool. Maven and Gradle examples are s
 </dependency>
 ```
 
+#### With MySQL Connector/J 6.x (development)
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>mysql-socket-factory-connector-j-6</artifactId>
+    <version>1.0.2</version>
+</dependency>
+```
+
 ### Gradle
 
+#### With MySQL Connector/J 5.x
 ```gradle
 compile 'com.google.cloud.sql:mysql-socket-factory:1.0.2'
 ```
 
-## Using
+#### With MySQL Connector/J 6.x
+```gradle
+compile 'com.google.cloud.sql:mysql-socket-factory-connector-j-6:1.0.2'
+```
+
+## Using the Socket Factory & Configuring JDBC Connection URL
 
 When specifying the JDBC connection URL, add two additional parameters:
 
 | Property         | Value         |
 | ---------------- | ------------- |
-| cloudSqlInstance | The instance connection name (which is found on the instance details page in Google Developers Console)  |
+| cloudSqlInstance | The instance ID (which is found on the instance details page in Google Cloud Console)  |
```

Comment:
Please change this back to "connection name", that is the terminology that is used in our UI and our docs.